tests_trung/utils/common.py

```diff
@@ -0,0 +1,21 @@
+from tests_trung.utils import constants
+import datetime
+from selenpy.support import browser
+from selenium.webdriver.remote.webelement import WebElement
+import random
+import string
+
+
+def get_today():
+    now = datetime.datetime.now()
+    return now.strftime(constants.DATE_FORMAT)
```

Comment:
Don't hard format here. When I need to get today in other format, I need to create new one?